orm/index_test.go

```diff
@@ -636,10 +645,27 @@ func testIndexImplementation(t *testing.T, newIdx func(MultiKeyIndexer) Index) {
 					t.Errorf("%d keys call returned unexpected keys for value %q", i, k.value)
 				}
 			}
+
+			// Write so that the data is persisted in the database.
+			// This is a mandatory step, because in memory
+			// implementation is more permissive.
+			if err := db.Write(); err != nil {
+				t.Fatalf("cache wrap write: %s", err)
+			}
 		})
 	}
 }
 
+func commitKVStore(t testing.TB) (db weave.CommitKVStore, cleanup func()) {
```

Comment:
I think this function belongs to test suite.

Reply:
What do you mean by test suite? This function is available only for tests and it is not visible for the non-test code.